tests/test_flows.py

```diff
@@ -133,6 +130,18 @@ def check_cmdu_type_single(
             return None
         return cmdus[0]
 
+    def check_no_cmdu_type(
+        self, msg: str, msg_type: int, eth_src: str, eth_dst: str = None
+    ) -> [sniffer.Packet]:
+        '''Like check_cmdu_type, but check that *no* machting CMDU is found.'''
```

Comment:
matching*

Why would you use this function over 
```
x = self.get_cmdu_capture_type(msg, msg_type, eth_src, eth_dst)
self.assertEqual(len(x), 0, "Got CMDU where none expected")
```
?
And what's the point of returning the result? IIRC `self.fail()` will raise an AssertionError, skipping the rest of the code in the function

Reply:
It's (currently) not using the unittest module, so there's no assertEqual etc. fail is a custom function in TestFlows.

In many cases, it's convenient to not abort right away on failure - that's why in googletest we usually use `EXPECT_*` rather than `ASSERT_*`. In particular, I often have a few failing checks while developing and then I'd prefer the test to continue rather than abort. Thus, I'm personally not very keen on moving to unittest.